kratos/geometries/geometry_shape_function_container.h

```diff
@@ -379,7 +381,8 @@ class GeometryShapeFunctionContainer
 
     virtual void load( Serializer& rSerializer )
     {
-        KRATOS_ERROR << "load function for geometry_shape_function_container not yet implemented." << std::endl;
+        rSerializer.save("default_method", default_method);
```

Comment:
```suggestion
        rSerializer.load("default_method", default_method);
```

Reply:
thanks, this is of course a bit crucial... ;)